app/dashboard/organization/grants/page.tsx

```diff
@@ -1,64 +1,143 @@
-      </div> 
-      :
-      <section className="animate-in">
-        {grantList.length === 0 ? 
-        <div className="bg-blue-100 border-t-4 border-blue-500 text-blue-700 px-4 py-3 mt-4 2xl:w-1/3 w-1/2">
-          <p className="font-semibold">You currently do not have any grants open.</p>
-          <p className="text-sm">Create a new grant below</p>
+      <PageHeading
+        header="Grants"
+        description="Grants your organization has open for applications or to be reviewed"
+      />
+      {loading ? (
+        <div className="text-center 2xl:mt-72 mt-52 text-darkmaroon">
+          <div className="text-xl font-medium">Retrieving grants</div>
+          <div className="loadingAnimation">
+            <div></div>
+            <div></div>
+            <div></div>
+          </div>
         </div>
-        :
-        <div className="flex flex-wrap mt-10 space-around">
-          {grantList.map((grant)=>{
-            return(<GrantCard key={grant.id} grantID={grant.id} name={grant.name} amount={grant.amount} description={grant.description} createdDate={grant.created_at}/>);
-          })}
-        </div>}
-
-        
-        <Link href="/dashboard/organization/grants/createGrant" className="inline-block text-darkmaroon border-2 border-darkmaroon rounded-md hover:bg-darkmaroon/20 py-1 px-2 mt-24 2xl:text-lg font-medium">
-          Create New Grant
-        </Link>
-      </section>}
+      ) : (
+        <>
+          {/* Open Grants */}
+          <section className="animate-in">
+            {grantList.length === 0 ? (
+              <div className="bg-blue-100 border-t-4 border-blue-500 text-blue-700 px-4 py-3 mt-4 2xl:w-1/3 w-1/2">
+                <p className="font-semibold">
+                  You currently do not have any grants open.
+                </p>
+                <p className="text-sm">Create a new grant below</p>
+              </div>
+            ) : (
+              <>
+                <h2 className="text-xl mt-10 border-b-2 border-body pb-2 w-[30rem]">
+                  Grants currently open
+                </h2>
+                <div className="flex flex-wrap mt-4 space-around">
+                  {grantList.map((grant) => {
+                    return (
+                      <GrantCard
+                        key={grant.id}
+                        grantID={grant.id}
+                        name={grant.name}
+                        amount={grant.amount}
+                        description={grant.description}
+                        createdDate={grant.created_at}
+                      />
+                    );
+                  })}
+                </div>
+              </>
+            )}
 
+            <Link
+              href="/dashboard/organization/grants/createGrant"
+              className="inline-block text-darkmaroon border-2 border-darkmaroon rounded-md hover:bg-darkmaroon/20 py-1 px-2 mt-8 2xl:text-lg font-medium"
+            >
+              Create New Grant
+            </Link>
```

Comment:
I think it'd be better to move the create new grant button below grants to be reviewed instead of in between open and in review grants